src/frametree.cpp

```diff
@@ -180,7 +180,7 @@ int FrameTree::removeFrameCommand() {
         pp->replaceChild(parent, newparent);
     } else {
         // if parent was root frame
-        root_ = newparent;
+        replaceNode(root_, newparent);
```

Comment:
Actually, we don't need to get `pp` at all. It should be sufficient to do
```
replaceNode(parent, newparent)
```
instead of the entire `if (pp) { … } else { … }` (and so we don't need the definition of `pp` at all)

Reply:
I changed it, still does not work (I added an assertion and a test that triggers it)